llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -1908,6 +1940,19 @@ void AArch64FrameLowering::emitPrologue(MachineFunction &MF,
     emitCalleeSavedGPRLocations(MBB, MBBI);
     emitCalleeSavedSVELocations(MBB, MBBI);
   }
+
+#ifndef NDEBUG
```

Comment:
We've got a few early returns, maybe wrap this in `make_scope_exit`.

Reply:
Done, thanks! This also keeps the code nicely together